translator/translate/logs/logs_collected/files/collect_list/collect_list_test.go

```diff
@@ -611,3 +611,115 @@ func TestLogFilters(t *testing.T) {
 	}}
 	assert.Equal(t, expectVal, val)
 }
+
+func TestRetention(t *testing.T) {
```

Comment:
I recommend making the name more specific:
`func TestRetentionDifferentLogGroups`